tests/stress_tests/CMakeLists.txt

```diff
@@ -9,12 +9,19 @@ if (CMAKE_BUILD_TYPE STREQUAL "")
     set(CMAKE_BUILD_TYPE "Release")
 endif()
 
-# Define directory where artifacts will be placed
-set(OUTPUT_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
+set(OpenVINO_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../")
 
-find_package(InferenceEngineDeveloperPackage REQUIRED)
+# Search OpenVINO Inference Engine installed
+find_package(InferenceEngine)
 
-set(OpenVINO_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../")
+if(NOT InferenceEngine_FOUND)
+    # Define directory where artifacts will be placed
+    set(OUTPUT_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
+
+    # Search OpenVINO Inference Engine via InferenceEngineDeveloperPackage_DIR
+    # in order to provide backward compatibility with old OpenVINO packages
+    find_package(InferenceEngineDeveloperPackage REQUIRED)
```

Comment:
why do we need it? we don't need to provide any compatibility for tests

Reply:
Thanks! Deleted